dev-libs/qtcompress/qtcompress-0_p20180513.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils toolchain-funcs virtualx
+
+DESCRIPTION="A public API for QZip in an easy to use module"
+
+HOMEPAGE="https://github.com/nezticle/qtcompress"
+MY_COMMIT="23f8831826cd72aedf99fc3699148b6c994fd677"
+SRC_URI="https://github.com/nezticle/qtcompress/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
```

Comment:
Are you absolutely 100 % percent sure this is needed? Note that because this is a snapshot, it'd change with every commit you snapshot into ebuild pretty much. That doesn't sound right...

Reply:
It's because it's a snapshot… I can't guarantee they won't change ABI, or even API, because they have no releases.

Reply:
The soname it uses, is the same as of Qt version it linked to

Reply:
What about making it 0_p20180513_p5_p14_p2:5.14.2? Then it needs to be updated with every qt update

Reply:
Sounds good to me. Does this work with Qt-5.15?

Reply:
Yes, but for the subslotted version like that, it would need to depend on the exact Qt version

Reply:
I think I got confused somewhere. This will be much simpler.